source/_docs/configuration/remote.markdown

```diff
@@ -30,3 +42,71 @@ If you cannot access your Home Assistant installation remotely, remember to chec
 Just putting a port up is not secure. You should definitely consider encrypting your traffic if you are accessing your Home Assistant installation remotely. For details please check the [set up encryption using Let's Encrypt](/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) blog post or this [detailed guide](/docs/ecosystem/certificates/lets_encrypt/) to using Let's Encrypt with Home Assistant.
 
 </div>
+
+
+## Apache Proxy
```

Comment:
All reverse proxy documentation has been removed from our end-user documentation. You can find, contribute or create them on the Community Guides section of the Community forums.

Please remove this section.

Reply:
Nabu Casa business decision?

Reply:
No, a Home Assistant project decision. As we became documentation for about every reverse proxy out there. A reverse proxy is optional, and not a core task of the project. Hence, moved out of the user documentation.

Reply:
There is a warning which states the current documentation is not recommended for use.  For this reason, this documentation is not useful.